html/elements/link.json

```diff
@@ -0,0 +1,1283 @@
+                  "ie_mobile": {
+                    "version_added": null
+                  },
+                  "opera": {
+                    "version_added": null
+                  },
+                  "opera_android": {
+                    "version_added": null
+                  },
+                  "safari": {
+                    "version_added": null
+                  },
+                  "safari_ios": {
+                    "version_added": null
+                  }
+                },
+                "status": {
+                  "experimental": true,
+                  "standard_track": true,
+                  "deprecated": false
+                }
+              },
+              "prerender": {
+                "support": {
+                  "webview_android": {
+                    "version_added": null
+                  },
+                  "chrome": {
+                    "version_added": null
+                  },
+                  "chrome_android": {
+                    "version_added": null
+                  },
+                  "edge": {
+                    "version_added": null
+                  },
+                  "edge_mobile": {
+                    "version_added": null
+                  },
+                  "firefox": {
+                    "version_added": null
+                  },
+                  "firefox_android": {
+                    "version_added": null
+                  },
+                  "ie": {
+                    "version_added": null
+                  },
+                  "ie_mobile": {
+                    "version_added": null
+                  },
+                  "opera": {
+                    "version_added": null
+                  },
+                  "opera_android": {
+                    "version_added": null
+                  },
+                  "safari": {
+                    "version_added": null
+                  },
+                  "safari_ios": {
+                    "version_added": null
+                  }
+                },
+                "status": {
+                  "experimental": true,
+                  "standard_track": true,
+                  "deprecated": false
+                }
+              },
+              "preconnect": {
+                "support": {
+                  "webview_android": {
+                    "version_added": "46.0"
+                  },
+                  "chrome": {
+                    "version_added": "46.0"
+                  },
+                  "chrome_android": {
+                    "version_added": "42.0"
+                  },
+                  "edge": {
+                    "version_added": false
+                  },
+                  "edge_mobile": {
+                    "version_added": false
+                  },
+                  "firefox": {
+                    "version_added": "39",
+                    "notes": "Before Firefox 41, it didn't obey the crossorigin attribute."
```

Comment:
Lets put crossorigin in `<code>` and use present tense.
`"notes": "Prior to Firefox 41, the <code>crossorigin</code> attribute isn't obeyed."`